docs/MachineConfigDaemon.md

```diff
@@ -14,6 +14,8 @@
 
 MachineConfigDaemon is scheduled on the machines in a cluster as a DaemonSet. This daemon is responsible for performing machine updates in OpenShift 4. The update will include tasks related to the systemd units, files on disk, operating system upgrades etc. The MachineConfigDaemon updates a machine to configuration defined by MachineConfig as instructed by the MachineConfigController.
 
+The MachineConfigDaemon is also responsible for annotating a node with `machineconfiguration.openshift.io/rhcosSSH=accessed` when it detects an SSH access to the machine.
```

Comment:
I know this is a bikeshed but "rhcosSSH" looks awkward.  How about just `machineconfiguration.openshift.io/ssh=accessed` ?

Reply:
Either works from my POV.

Reply:
Sure, will change